aiidalab/home.py

```diff
@@ -123,3 +138,49 @@ def move_updown(self, name, delta):
         order.insert(j, name)
         config['order'] = order
         self.write_config(config)
+
+
+class AppWidget(ipw.VBox):
```

Comment:
Can you remove the rectangular frame from around the home app?

Reply:
Yes.

Reply:
done